terraform/modules/k8s/airflow/main/templates/airflow.yaml

```diff
@@ -42,35 +53,40 @@ airflow:
           name: airflow-postgres
           key: postgresql-password
 
+  usersUpdate:
+    enabled: false
```

Comment:
Why did you disable users update?

Reply:
fixed

Reply:
As final decision, we disabled usersUpdate and set airflow.users to empty array